lookout/style/format/model.py

```diff
@@ -21,9 +21,16 @@ class FormatModel(AnalyzerModel):
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
         self._rules_by_lang = {}  # type: Dict[str, Rules]
+        self._config = {}  # type: Dict[str, Any]
+
+    def construct(self, analyzer: Type["Analyzer"], ptr: ReferencePointer, config: Dict[str, Any]
+                  ) -> "FormatModel":
+        super().construct(analyzer, ptr)
+        self._config = config
```

Comment:
The config should be different per language => we need to include it inside Rules.
In other words: Rules instance must contain all the configuration and there is no global one. Things like line filtering length are the parts of Rules config.

BTW - we should fix train() because currently it does not distinguish between languages.

Reply:
also do not get about `train`.

Reply:
`train()` has the same config for all the languages. Just checked the code - still does. It shouldn't.

Reply:
I like this config thing @m09 proposes.